src/oaklib/implementations/pronto/pronto_implementation.py

```diff
@@ -836,10 +836,16 @@ def apply_patch(
             for syn in t.synonyms:
                 if syn.description == patch.old_value:
                     syn.description = patch.new_value
+        elif isinstance(patch, kgcl.NewTextDefinition):
+            t = self._entity(patch.about_node, strict=True)
+            xrefs = t.definition.xrefs if t.definition else []
+            v = f'"{patch.new_value}"'
```

Comment:
This isn't correct. For the pronto adapter you are using the pronto data model directly, so don't quote

Reply:
Isn't this is similar to `simple_obo_implementation`  since this is in `apply_patch()` specific to the `kgcl.NewTextDefinition` object which is what we need?